api/src/main/java/org/pmiops/workbench/dataset/DataSetService.java

```diff
@@ -37,11 +44,13 @@ DbDataset cloneDataSetToWorkspace(
 
   List<DbCohort> getCohortsForDataset(DbDataset dataSet);
 
-  List<DbDataset> getDataSets(ResourceType resourceType, long resourceId);
+  List<DataSet> getDataSets(ResourceType resourceType, long resourceId);
```

Comment:
Should this also be Optional?

Reply:
Not sure return type of `List` needs to be `Optional`... it either has items on it or it's empty... no chance of a `NullPointer`